Dockerfile

```
@@ -24,4 +24,8 @@ RUN poetry config virtualenvs.create false && \
 
 COPY . /app/
 
+RUN useradd -r -u 1000 -g root appuserRunnerManager \
+    && chown -R appuserRunnerManager /app
+USER appuserRunnerManager
+
```

First let's keep the name short

```suggestion
RUN useradd -r -u 1000 -g root runner-manager \
    && chown -R runner-manager /app
USER runner-manager

```

Second the important thing of setting a user is to remove any sort of root access. I'm not 100% sure on what the `-g root` option those, but if it adds root access to the image it should be removed.

And if you need root access review why, might be because you are using a 80 port, you can change it to 8080 to sort this out.

---

You're right, I will change the `-g root` option. I will try without changing the port first, then I will see if it works or no